docs/modules/ROOT/partials/proc-persistence-kafkasql-plain.adoc

```diff
@@ -62,14 +62,23 @@ status:
   ...
 ----
 
-. Click the *Kafka Topic* tab, and then *Create Kafka Topic* to create the required `global-id-topic`:
+. The default Kafka topic name that {registry} uses to store data is `kafkasql-journal`.
+This topic is created automatically by {registry}.
+You can override this behavior or the default topic name by setting the appropriate environment variables (default values):
+ ** `REGISTRY_KAFKASQL_TOPIC_AUTO_CREATE=true` and
+ ** `REGISTRY_KAFKASQL_TOPIC=kafkasql-journal`.
```

Comment:
Suggest removing `and` + `.` from end of bullets, i.e., use only:
** `REGISTRY_KAFKASQL_TOPIC_AUTO_CREATE=true` 
** `REGISTRY_KAFKASQL_TOPIC=kafkasql-journal`